doc/service_interface.md

```diff
@@ -0,0 +1,70 @@
+# Radar Service Interface
+
+The Service Interface is a simple HTTP-based interface for services to query radar.
+
+## Service Endpoint
+
+- http(s)://hostname:port/radar/service
+
+## GET
+
+E.g. `http://localhost:8000/radar/service?to=status:/segment/foobar`
+
+### Querystring Parameters
+- `to` : the full resource scope, e.g. `status:/segment/foobar`
+
+This is a simplified case of the POST endpoint for Radar resources supporting `get` operations.
+
+### Response
+A `get server message` depending on the resource type, e.g. for a Status:
+
+```json
+{
+  "op": "get",
+  "to": "status:/account/clients/1452556889570",
+  "value": {
+    "1452556889570": "sdf"
+  }
+}
+```
+
+or for a Presence:
+
+```json
+{
+  "op": "get",
+  "to": "presence:/account/box1",
+  "value": {
+    "1452560641403": 2
```

Comment:
The message flow does not change in this PR, so the response will be the v2 structure if there is a message.options.version == 2 set on the incoming message. The flow is:
ServerInterface -> Server#_processMessage -> Server#_handleResourceMessage -> dispatch to Presence resource -> Presence#get -> Presence#manager.fullRead (which then switches based on the message.options.version)

In other words, it's still up to the client to specify the response version for Presence resources. Cleaning up this behavior I think is outside the scope of this PR and would require careful attention to avoid breaking current scenarios.

e.g.
```sh
> curl -X POST http://localhost:8000/radar/service -H "content-type: application/json" -d '{"op":"get", "to":"presence:/presents/box1", "options":{"version":1}}' -s | jq
{
  "op": "get",
  "to": "presence:/presents/box1",
  "value": {
    "1454536247917": 2
  }
}

> curl -X POST http://localhost:8000/radar/service -H "content-type: application/json" -d '{"op":"get", "to":"presence:/presents/box1", "options":{"version":2}}' -s | jq
{
  "op": "get",
  "to": "presence:/presents/box1",
  "value": {
    "1454536247917": {
      "clients": {
        "zmQ8Rjkr2Xf1XqV2AAAA": {}
      },
      "userType": 2
    }
  }
}
```